### PR TITLE
Move common code for upload new sound to utils

### PR DESCRIPTION
**Pull request by andrebola**
issue #799

### CODE REVIEW

File: sounds/views.py [hunk 1]
```diff
@@ -461,6 +461,7 @@ def pack_edit(request, username, pack_id):
     if request.method == "POST":
         form = PackEditForm(request.POST, instance=pack)
         if form.is_valid():
+            pack.sound_set.all().update(is_index_dirty=True)
```
Comment (ffont) [inline sounds/views.py]: Why is this change here? Isn't this called wight below?

File: sounds/management/commands/csv_bulk_upload.py [hunk 2]
```diff
@@ -124,106 +124,41 @@ def handle(self, *args, **options):
...
-                sound.pack.process()
+            except utils.sound_upload.NoAudioException:
+                continue
+            except utils.sound_upload.AlreadyExistsException:
+                print 'The file %s is already part of freesound, not uploading it' % (dest_path,)
+                continue
+            except utils.sound_upload.CantMoveException as e:
+                print e.message
 
-            print 'Successfully uploaded sound ' + sound.original_filename
```
Comment (ffont) [inline sounds/management/commands/csv_bulk_upload.py]: Why did you remove the print? Is is somewhere else?

**Comment by ffont** [comment]
@andrebola  I left a couple of comments.
It looks good but I'd like to be 100% this works well.

For example, is the new `is_explicit` thing considered here? (I guess not as the two PR are parallel)
We have a test of the `create_sound` util function and for the view to upload sounds (https://github.com/MTG/freesound/blob/master/accounts/tests.py#L549), but we are not testing the the apiv2 endpoint and the the csv bulk upload . I guess if we don't add the `is_explicit` field then by default `is_explicit` is set to False and that's it (which is OK), but do the other stuff work? I wanted to test manually but this branch now conflicts with master and I'm not sure how to fix it. Can you fix the conflicts so I can manually test the PR? It would be also good to add those missing tests although it might be more complex...

File: sounds/management/commands/csv_bulk_upload.py (hunk 2 above)
Comment (andrebola) [inline sounds/management/commands/csv_bulk_upload.py]: yes, it's on line 155

**Comment by andrebola** [comment]
Merged with Master added is_explicit.  I think we don't have much tests for the api in general.
